src/Driver/AbstractDB2Driver.php

```diff
@@ -2,29 +2,28 @@
 
 namespace DoctrineDbalIbmi\Driver;
 
+use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\Driver;
-use Doctrine\DBAL\Platforms\DB2Platform;
 use DoctrineDbalIbmi\Platform\DB2IBMiPlatform;
 use DoctrineDbalIbmi\Schema\DB2IBMiSchemaManager;
-use DoctrineDbalIbmi\Schema\DB2LUWSchemaManager;
 
 abstract class AbstractDB2Driver implements Driver
 {
     const SYSTEM_IBMI = 'AIX';
     const SYSTEM_IBMI_OS400 = 'OS400';
 
     /**
-     * @return bool
+     * @return true
      */
     public static function isIbmi()
```

Comment:
If we're only going to support IBM i, then:

* If there are differences between LUW/i ibm_db2 (and perhaps to a lesser degree, ODBC) even with an IBM i server, it may be worth keeping. Even with ODBC, could i.e. automatically dispatch to `*LOCAL` if appropriate.
* If there aren't any, we should remove this function and its constants.

Reply:
I had marked this method as deprecated before, but I removed the deprecation based on a suggestion. I didn't remove it because a possible unintended BC break.

>If there are differences between LUW/i ibm_db2 (and perhaps to a lesser degree, ODBC) even with an IBM i server, it may be worth keeping. Even with ODBC, could i.e. automatically dispatch to *LOCAL if appropriate.

In that case, I think it should be renamed to something like `isRunningLocally()` or something similar, because this library can be used even remotely to connect with a *IBM DB2 for i* database (in fact, this is my current use case).